src/parser/content-evaluator-module.ts

```diff
@@ -137,7 +142,7 @@ export class ContentEvaluatorModule implements Module {
     const maxTokens = this._calculateMaxTokens(dummyResponse);
 
     const response: OpenAI.Chat.ChatCompletion = await this._openAi.chat.completions.create({
-      model: "gpt-4o-2024-08-06",
+      model: this._configuration?.openAi.model || "gpt-4o-2024-08-06",
```

Comment:
Wouldn't the typebox default kick in here if they didn't specify a model anyway?